Unreal/CarlaUE4/Plugins/Carla/Source/Carla/OpenDriveActor.h

```diff
@@ -43,6 +45,33 @@ class CARLA_API AOpenDriveActor : public AActor
   UPROPERTY()
   TArray<AVehicleSpawnPoint *> VehicleSpawners;
 
+  UPROPERTY()
+  TArray<AActor *> PersistentTrafficLights;
+
+  UPROPERTY()
+  TArray<AActor *> PersistentTrafficSigns;
+
+  UPROPERTY()
+  TSubclassOf<class AActor> TrafficLightBlueprintClass;
```

Comment:
These are workarounds waiting for the final change into c++. Once the TrafficLights become a C++ class, everything will be easier